HopsanGenerator/templates/fmu_hopsan.cpp

```diff
@@ -199,14 +200,18 @@ void hopsan_set_real(int vr, double value)
         (*dataPtrs[vr]) = value;
     }
     else if(realParametersMap.count(vr)) {
-        spCoreComponentSystem->setParameterValue(realParametersMap[vr], hopsan::HString(std::to_string(value).c_str()));
+        std::ostringstream ss;
+        ss << value;
+        spCoreComponentSystem->setParameterValue(realParametersMap[vr], hopsan::HString(ss.str().c_str()));
```

Comment:
There is a hopsan::to_hstring() function that mimics std::to_string, at least for the most common types

Reply:
Good point, I forgot we had that one. Then we can get rid of <sstream> as well.